### PR TITLE
Floorbot settings description revised to explain it better

### PR DESCRIPTION
**Pull request by Oyu07**
## What Does This PR Do

Changes text on what Autotile and Replacetile does on floorbot to better explain
As it appears, no one really knows why any of the settings do other than the "fix stuff" and just enable everything, which you shouldnt really.

Includes revised text
Replace Floor Tiles ---> Add floor tiles on exposed hull plating

With tooltips

## Why It's Good For The Game

People do not know what the top 2 settings do, hopefully this will explain better

## Images of changes

yB91zHObze

Tus9aNePOT

## Testing

~~I did not~~
Nothing functional changed, check screenshots thats all

## Changelog
:cl:
tweak: Floorbot settings descriptions revised and added tooltip
/:cl:

### CODE REVIEW

File: tgui/packages/tgui/interfaces/BotFloor.js [hunk 1]
```diff
@@ -31,13 +31,15 @@ export const BotFloor = (props, context) => {
...
+            tooltip="Fixing a plating requires the removal of floor tile. This will place it back after repairing. Same goes for hull breaches"
             disabled={noaccess}
             onClick={() => act('autotile')}
           />
           <Button.Checkbox
             fluid
             checked={replace}
-            content="Replace floor tiles"
+            content="Add floor tiles on exposed hull plating"
+            tooltip="Example: It will add tiles to maintanence"
```
Comment (Henri215) [inline tgui/packages/tgui/interfaces/BotFloor.js:42]:
```suggestion
            tooltip="Example: It will add tiles to maintenance"
```

Comment (Oyu07) [reply tgui/packages/tgui/interfaces/BotFloor.js]: bundle coming soon^tm

**Comment by Oyu07** [comment]
!merge_upstream

**Comment by Oyu07** [comment]
!merge_upstream

**Comment by Oyu07** [comment]
!merge_upstream

**Comment by Oyu07** [comment]
lets try one more time maybe?

**Comment by Oyu07** [comment]
!merge_upstream

**Comment by Oyu07** [comment]
!merge_upstream

**Review by Henri215 — APPROVED** [review]
LGTM
I still think that a better name could be used too, but i cant think of one myself

**Comment by Contrabang** [comment]
!merge_upstream

**Review by Contrabang — APPROVED** [review]

**Comment by Contrabang** [comment]
Yeah it seems that the `!merge_upstream` doesn't work properly

**Comment by S34NW** [comment]
!merge_upstream

**Review by Contrabang — APPROVED** [review]

**Pull request merged**